sig-contributor-experience/charter.md

```diff
@@ -52,11 +44,15 @@
        * kubernetes-wg-contribex@
     * Announce at weekly Kubernetes Community Meeting on Thursdays
 
-
-
-## TODO:
-* Next iteration needs a breakdown of responsibilities by each leader
-* Provide some guidance, like a super lightweight proposal template. Or at least ask those making a proposal to consider things like: "Who does this change impact, and how? What can/will cease working as part of this change, and how does the proposer plan to help mitigate that?"
-* Add KEP proposal process info^?
-
-### thanks to those who have reviewed this doc and provided guidance already
+## Leadership Composition
+  * Two (2) Chairs and Two (2) Technical Leads unless group decides that more is needed
+  * Each Chair and Technical Lead will be responsible for a sub-group in addition to assisting with the processes outlined here. 
+  * Lazy Consensus within the SIG to make final decision 
+* Changing leadership
+  * Candidates are nominated through the mailing list and confirmed at SIG meeting; self nomination is fine. Candidate needs two +1’s from folks representing different companies or independent/freelancers in the community to be considered a nominee 
+  * Chair/TL may step down at any time for any reason and propose a replacement that is already an active contributing member of Contributor Experience. They will make the change to the OWNERs file
+  * A Chair/TL may be asked to step down if one of the following occurs:
+    * Inactivity as determined by the governance of the project or as specifically described in the ask to SIG leadership.
```

Comment:
I like trailing punctuation for list entries.  But since the pattern in these docs seems to be to not use trailing punctuation, you may want to drop the final period here.